model/event/event_finder.go

```diff
@@ -125,21 +125,24 @@ func CountUnprocessedEvents() (int, error) {
 // === Queries ===
 
 // Host Events
-func MostRecentHostEvents(id string, tag string, n int) db.Q {
+func MostRecentHostEvents(id string, tag string, n int, sortAsc bool) db.Q {
 	filter := ResourceTypeKeyIs(ResourceTypeHost)
 	if tag != "" {
 		filter[ResourceIdKey] = bson.M{"$in": []string{id, tag}}
 	} else {
 		filter[ResourceIdKey] = id
 	}
 
+	if sortAsc {
+		return db.Query(filter).Sort([]string{TimestampKey}).Limit(n)
+	}
 	return db.Query(filter).Sort([]string{"-" + TimestampKey}).Limit(n)
```

Comment:
slightly wonder if it makes sense to write it like this
```
	sortMethod := []string{"-" + TimestampKey} // use different variable name if appropriate
	if sortAsc {
		sortMethod = []string{TimestampKey}
	}
	return db.Query(filter).Sort(sortMethod).Limit(n)
```

Reply:
Makes sense!